internal/services/springcloud/spring_cloud_java_deployment_resource.go

```diff
@@ -353,34 +353,18 @@ func resourceSprintCloudJavaDeploymentSchema() map[string]*pluginsdk.Schema {
 				Schema: map[string]*pluginsdk.Schema{
 					// The value returned in GET will be recalculated by the service if the deprecated "cpu" is honored, so make this property as Computed.
 					"cpu": {
-						Type:     pluginsdk.TypeString,
-						Optional: true,
-						Computed: true,
-						ValidateFunc: validation.StringInSlice([]string{
-							"500m",
-							"1",
-							"2",
-							"3",
-							"4",
-						}, false),
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
+						Computed:     true,
+						ValidateFunc: validation.StringIsNotEmpty,
```

Comment:
we should add a comment here explaining that additional values _may_ be possible via the Service Team - so we're intentionally not validating this